x/staging/posts/keeper/keeper_reports.go

```diff
@@ -32,22 +45,20 @@ func (k Keeper) SaveReport(ctx sdk.Context, report types.Report) error {
 // GetPostReports returns the list of reports associated with the given postID.
 // If no report is associated with the given postID the function will returns an empty list.
 func (k Keeper) GetPostReports(ctx sdk.Context, postID string) []types.Report {
-	store := ctx.KVStore(k.storeKey)
-	return types.MustUnmarshalReports(store.Get(types.ReportStoreKey(postID)), k.cdc)
+	var reports []types.Report
+	k.IteratePostReportsByPost(ctx, postID, func(_ int64, report types.Report) bool {
+		reports = append(reports, report)
+		return false
+	})
+	return reports
 }
```

Comment:
I think this should be moved inside the `alias_functions` file together with similar methods

Reply:
I think we should leave it here since it's a reports' related method